mvlearn/compose/rsm.py

```diff
@@ -42,15 +42,15 @@ class RandomSubspaceMethod(TransformerMixin):
 
     References
     ----------
-    .. [#1RSM] Ho, Tin Kam. "The random subspace method for constructing
-            decision forests." IEEE transactions on pattern analysis
-            and machine intelligence 20.8 (1998): 832-844.
+    .. [#1RSM] Tin Kam Ho. "The random subspace method for constructing
+            decision forests." IEEE trans-actions on pattern analysis and
```

Comment:
```suggestion
            decision forests." IEEE transactions on pattern analysis and
```

Reply:
actually the recommended way is : IEEE Transactions on Pattern Analysis and Machine Intelligence
with caps